src/aleph/vm/orchestrator/views/authentication.py

```diff
@@ -55,6 +56,7 @@ class SignedPubKeyPayload(BaseModel):
     # alg: Literal["ECDSA"]
     address: str
     expires: str
+    chain: Literal["ETH"] | Literal["SOL"] = "ETH"
```

Comment:
Should this use an Enum ? For example `aleph_message.models.Chain.ETH` and ``aleph_message.models.Chain.SOL` ?